src/libdecaf/src/modules/coreinit/coreinit_fastmutex.cpp

```diff
@@ -193,46 +193,59 @@ fastMutexHardUnlock(OSFastMutex *mutex)
    internal::unlockScheduler();
 }
 
+static void
+OSFastMutex_Unlock_common(OSFastMutex *mutex) {
```

Comment:
This should probably be NoLock postfixed?

Reply:
Additionally, OSFastMutex_Unlock_one might be more in line with our standards, and makes it more clear what it does.

Reply:
Well, you tell me ;) I don't know about your naming conventions and was not sure whether NoLock was directed at the mutex itself or the scheduler.